final release/format_data.py

```diff
@@ -0,0 +1,208 @@
+
+# coding: utf-8
+
+# In[45]:
+
+
+import numpy as np
+import pickle
```

Comment:
Split imports into

- standard (`pickle`, `sys`, `os`)
- library (the others)
- repo (you don't have any of these yet)

And alphabetise them.

Reply:
is h5py not standard?

Reply:
No.

Reply:
https://docs.python.org/3/library/